Tophat/Models/AppleApplication.swift

```diff
@@ -68,8 +67,24 @@ struct AppleApplication: Application {
 	}
 
 	var platform: Platform {
-		// Only iOS supported, for now.
-		.iOS
+		guard let platformNames = bundle?.infoDictionary?["CFBundleSupportedPlatforms"] as? [String] else {
```

Comment:
At first I thought this was Tophat's supported platforms, which would only be `macOS`, but I see now it's the downloaded app's supported platforms. 👌

Reply:
Yeah, we create a `Bundle` for the downloaded app to read it as well.